RadixWallet/Clients/SecureStorageClient/SecureStorageClient+Interface.swift

```diff
@@ -11,6 +11,7 @@ public struct SecureStorageClient: Sendable {
 	public var loadProfileSnapshotData: LoadProfileSnapshotData
 	public var loadProfileSnapshot: LoadProfileSnapshot
 	public var loadProfile: LoadProfile
+	public var deleteProfile: DeleteProfile
```

Comment:
So we already have a `DeleteProfileAndMnemonicsByFactorSourceIDs`, which actually is something as `DeleteProfileAndMnemonicsByProfileId`. 
Do we actually want to ONLY delete the Profile and keep the mnemonics on some scenario?

Reply:
This one I don't know, @CyonAlexRDX will tell us what is needed.

Reply:
It is more a product owner question... But I think we should NEVER delete mnemonics without user knowing it. So safest to just delete profile and keep mnemonics. 

This can, however, lead to "dangling mnemonics", given current impl in Swift. We we ought to later see if we can changed that, because we only ever do the query "load EVERYTHING" in the `DebugKeychainContents` debug screen. Since otherwise, it is only a profile knowing the exact lookup-key (`FactorSourceID`) of a FactorSource (Mnemonic).

Reply:
For context, this is called after migrating the profile backup to icloud.

Reply:
but if it isn't used on new Security Center, why are we exposing the `deleteProfile` as public?

Reply:
It is used [in CloudBackupClient](https://github.com/radixdlt/babylon-wallet-ios/pull/1106/files#diff-c4ae62b3f1ebe8cd50673c681cd202bc3b22760a37e90016f7781f9fd94acdc8R126)

Reply:
so we need to consult with product if it is desired to keep mnemonics on that delete?